src/main/java/logger/controllers/MessageController.java

```diff
@@ -0,0 +1,21 @@
+package logger.controllers;
+
+import logger.Message;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.Collection;
+
+@RestController
+public class MessageController {
+    @RequestMapping(value = "messages")
+    public Collection<Message> messages() {
+        return new ArrayList<Message>() {{
```

Comment:
oh my god. use `List.of(new Message(...), new Message(...), ...)`.

Reply:
didnt know i can do something like this 👍